sickbeard/search.py

```diff
@@ -41,20 +38,22 @@
 from sickbeard import ui
 from sickbeard import failed_history
 from sickbeard import common
+
 from sickrage.helper.encoding import ek
 from sickrage.helper.exceptions import AuthException, ex
 from sickrage.providers.GenericProvider import GenericProvider
 from sickrage.helper.common import enabled_providers
 
+from socket import timeout as SocketTimeout
+
 
 def _downloadResult(result):
     """
-    Downloads a result to the appropriate black hole folder.
+    Download a result to the appropriate black hole folder.
```

Comment:
Is there a point in removing `s` and `es` from all the conjugated verbs?

`( The method / It ) Downloads a result to the appropriate black hole folder.`

I'm just asking since I saw @labrys doing the same... I'm a java guy and when working with java, all APIs (including the official java API) uses the verbs in the 3rd person as you can see it [here](http://docs.oracle.com/javase/8/docs/api/java/util/Collections.html)